lib/tdf3/src/client/builders.ts

```diff
@@ -794,8 +794,25 @@ class DecryptParamsBuilder {
   /**
    * @param rcaParams
    */
-  setRcaSource(rcaParams: RcaParams) {
-    this._params.rcaSource = rcaParams;
+  setRcaSource(rcaParams: RcaParams | RcaLink) {
```

Comment:
Is this a copy of the existing builder method to maintain compatibility? If this is not, I'd suggest moving the logic out into a `urlToRcaParams` function and export that, to allow for better tree shaking and more clear errors - generally I don't like setters to throw